requirements.txt

```diff
@@ -20,6 +20,7 @@ profilehooks==1.9.0
 PyGithub==1.35
 PyNMA==1.0
 python-dateutil==2.6.1
+reflink=0.1.4
```

Comment:
Double `==` instead of single `=`. You also need to include the reflink package in the `ext` folder, as we currently still bundle external packages.